app/js/components/systemMetadata/systemMetadata.directive.js

```diff
@@ -110,12 +103,12 @@ function systemMetadataCtrl(
             const data = find(system_metadata, {category: category});
 
             if (data && !data.noData) {
-                if ($scope.tableData) {
-                    $scope.tabs[$scope.tableData.category] = false;
+                if (currentTab !== category) {
+                    $scope.tabsContent[currentTab] = null;
```

Comment:
just curious - why set tabsContent[currentTab] to null? Would it be better to leave data in tabsContent so that next time the user opens that tab again we can skip the `find` and just serve "cached" tab content?

Reply:
@jharting you are right, I should have done that to begin with.